clang/test/Analysis/NewDelete-intersections.mm

```diff
@@ -58,14 +60,14 @@ void testFreeOpNew() {
   void *p = operator new(0);
   free(p);
   // mismatch-warning@-1{{Memory allocated by 'operator new' should be deallocated by 'delete', not 'free()'}}
-} // leak-warning{{Potential leak of memory pointed to by 'p'}}
+}
```

Comment:
To be clear, these `leak-warning` s were added during WIP stage of this PR and then I dropped it after I added invalidation of freed regions.

I will anyway check if it is be possible to add them, tho

Reply:
> To be clear, these leak-warning s were added during WIP stage of this PR

Oops, I was only looking at the "changes since last review" diff and didn't notice this. Based on this, I'd lean toward preserving the old behavior to make this commit closer to being an NFC. (But I'd support adding this leak warning as a separate commit, if you're interested.)